bokehjs/src/lib/core/types.ts

```diff
@@ -4,8 +4,8 @@ export type Color = string
 
 export {TypedArray} from "./util/ndarray"
 
-export type NumberArray = Float64Array
-export const NumberArray = Float64Array
```

Comment:
This seems like it should also be maintained under test.

Reply:
Yes, but not as a `true == true` type test. What you want to do, is to design tests that would let us know about the effect on bokehjs from changing such constant (with real numbers and tolerances). Right now there are a few test cases that fail, but all of them are coincidental, so not good at all.

Reply:
I'm not sure I agree with that at all. The value of this type alias is a policy decision for us to make as a project. We only need to research the effect of different values as inputs to that decision. The decision having been made, the most important priority is to maintain the policy protect against accidental change to it. It's certainly possible that could be done implicitly by observing side-effects of a changed value, but that IMO that obscures the intent.